components/Datepicker/react.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import classNames from 'classnames/bind'
+import DatePicker, { CalendarContainer } from 'react-datepicker'
+import styles from './style.module.scss'
+import 'react-datepicker/dist/react-datepicker.css'
+
+const cx = classNames.bind(styles)
+
+const Datepicker = ({ className, ...props }) => {
```

Comment:
The Datepicker className is prop is unused. If you meant to just pass all props into the 3rd party DatePicker, you'll want to remove className here and just do `const Datepicker = props =>`